src/status_im/bots/models.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.bots.models)
```

Comment:
agree, simpler to keep it in events.cljs imo - that's the only place these functions are used as far as I can tell.